mlflow/store/file_store.py

```diff
@@ -48,6 +48,10 @@ def _make_persisted_run_info_dict(run_info):
     run_info_dict = dict(run_info)
     run_info_dict['tags'] = []
     run_info_dict['name'] = ''
+    if 'status' in run_info_dict:
+        run_info_dict['status'] = RunStatus.from_string(run_info.status)
```

Comment:
e.g., for old file store data, this may be a number?

Reply:
yes. `FileStore` writes status as enum, where as `SqlAlchemyStore` writes it as a string. This is weird, but the APIs out of these stores are `RunInfo`